openml/study/functions.py

```diff
@@ -4,44 +4,49 @@
 import openml._api_calls
 
 
-def _multitag_to_list(result_dict, tag):
-    if isinstance(result_dict[tag], list):
-        return result_dict[tag]
-    elif isinstance(result_dict[tag], dict):
-        return [result_dict[tag]]
-    else:
-        raise TypeError()
-
-
-def get_study(study_id, type=None):
-    '''
+def get_study(study_id, knowledge_type=None):
```

Comment:
Could you please document the arguments? I have no idea what `knowledge_type` is.